app/mailers/devise_mailer.rb

```diff
@@ -22,4 +22,51 @@ def password_change(record, opts={})
 
     mail(to: record.email)
   end
-end
\ No newline at end of file
+
+  def invitation_instructions(record, token, opts={})
+    case record.team_members.last.role
+
+    when 'admin'
+      set_template('cca89657-03e4-4c88-9c9b-affd4c406ce6')
```

Comment:
So perhaps `secrets` is the best place for these and we remove that file from version control.

What about encrypting them? http://edgeguides.rubyonrails.org/5_1_release_notes.html#encrypted-secrets